python-cli/mft_cli/airavata_mft_cli/bootstrap.py

```diff
@@ -160,6 +164,10 @@ def start_mft():
     zip_path = os.path.join(os.path.expanduser('~'), ".mft/Standalone-Service-0.01-bin.zip")
     download_and_unarchive(url, zip_path)
 
+  while not is_port_open(8500):
+    print("Waiting for Consul to start...")
+    time.sleep(1)
```

Comment:
@ShivangMishra Can you make this only run for 20 seconds and if the port is not open by then, throw an error message and exit the program?

Reply:
Yes sure. Done.